homeassistant/components/habitica/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: No events are registered by the integration.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: There is no options flow.
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: done
+  reauthentication-flow: todo
+  test-coverage: todo
+
+  # Gold
+  devices: done
+  diagnostics: done
+  discovery-update-info:
+    status: exempt
+    comment: Integration represents a service
+  discovery:
+    status: exempt
+    comment: Integration represents a service
+  docs-data-update: done
+  docs-examples: done
+  docs-known-limitations: done
+  docs-supported-devices:
+    status: exempt
+    comment: No supportable devices.
+  docs-supported-functions: done
+  docs-troubleshooting: todo
+  docs-use-cases: done
+  dynamic-devices:
+    status: exempt
+    comment: |
+      Integration is a service, no devices that could be added at runtime.
+      Button entities for casting skills are created/removed dynamically if unlocked or on class change
+  entity-category:
+    status: done
+    comment: Default categories are appropriate for currently available entities.
+  entity-device-class: done
+  entity-disabled-by-default: done
+  entity-translations: done
+  exception-translations: done
+  icon-translations: done
+  reconfiguration-flow: todo
+  repair-issues:
+    status: done
+    comment: Used to inform of deprecated entities and actions.
+  stale-devices:
+    status: done
+    comment: Not applicable. Only one device per config entry. Removed together with the config entry.
+
+  # Platinum
+  async-dependency: done
```

Comment:
Habitipy uses aiohttp for async requests. Only the class init is not thread-safe, is that the reason?

Reply:
Oh right, I saw that executor job and this was the first comment I made

Reply:
Let's set it to todo for now, since I know you are working on a replacement and creating an executor job on boot isn't really the epitome of integration quality IMO